includes/amp-helper-functions.php

```diff
@@ -54,6 +54,17 @@ function amp_deactivate( $network_wide = false ) {
  * @since 1.5
  */
 function amp_bootstrap_plugin() {
+	/**
+	 * Filters whether AMP is enabled on the current site.
+	 *
+	 * Useful if the plugin is network activated and you want to turn it off on select sites.
+	 *
+	 * @since 0.2
```

Comment:
I'd add a since tag for the breaking change. Maybe something like this:
```suggestion
	 * @since 0.2
	 * @since 2.0 Postponed to run at plugins_loaded:8 instead of right after file load.
```